pkg/splunk/enterprise/util.go

```diff
@@ -950,13 +947,16 @@ func HasAppRepoCheckTimerExpired(appInfoContext *enterpriseApi.AppDeploymentCont
 // GetNextRequeueTime gets the next reconcile requeue time based on the appRepoPollInterval.
 // There can be some time elapsed between when we first set lastAppInfoCheckTime and when the CR is in Ready state.
 // Hence we need to subtract the delta time elapsed from the actual polling interval,
-// so that the next reconile would happen at the right time.
+// so that the next reconcile would happen at the right time.
 func GetNextRequeueTime(appRepoPollInterval, lastCheckTime int64) time.Duration {
 	scopedLog := log.WithName("GetNextRequeueTime")
 	currentEpoch := time.Now().Unix()
 
 	var nextRequeueTimeInSec int64
 	nextRequeueTimeInSec = appRepoPollInterval - (currentEpoch - lastCheckTime)
+	if nextRequeueTimeInSec < 0 {
+		nextRequeueTimeInSec = 5
```

Comment:
minimum requeue time?

Reply:
When can this be negative?

Reply:
Correct, this is the default we use across the reconciliations, so using the same value here. I added this, just to make sure we are not getting into the -ve values when installing larger apps.

Reply:
Say, if the polling interval is 1 min, and If we are installing a large app like ES App, we are not setting the `LastAppInfoCheckTime` for as high as 15 minutes. In such cases, it helps.